Snippets/Snippets_3/DataBus/CleanupStrategy/Usage.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Snippets3.DataBus.CleanupStrategy
+{
+    using System;
+    using System.IO;
+    using Snippets3.DataBus.DataBusProperty;
+
+    public class Usage
+    {
+        #region FileLocationForDatabusFiles
+        public void Handle(MessageWithLargePayload message)
+        {
+            string filename = Path.Combine(@"c:\databus_files\", message.LargeBlob.Key);
```

Comment:
It was a question by a customer how to obtain the file location. I'm not surprised they can't figure it out by themselves, that's why I added it. And the fact that it's in "handle" method, is so they understand that it's the receiving side.

And hopefully they understand that removing files in the handler isn't smart, due to the story they just read. But I could add something like "// Execute cleanup strategy, like scheduling the delete after x-days"